### ISSUE TITLE
Is the `gaproot/pkg/` symlink a problem for this action?

### ISSUE THREAD

**Issue by wilfwilson** [gap-actions/process-coverage#13]
`gcov` searches the package directory for coverage files... but the package directory contains itself because of the symlink in `gaproot/pkg/`.

Does that mean `gcov` is wasting loads of effort finding the same files over and over again?

Should the `gaproot/pkg/` symlink be deleted by this action perhaps, even if just temporarily?

**Comment by wilfwilson** [comment]
Maybe it's more a problem for `codecov/codecov-action@v2` - e.g. we get lines like:
```
[2022-01-28T14:18:08.451Z] ['info'] Processing /home/runner/work/Digraphs/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/coverage/UtVACY.coverage.22151...
```
which is a bit ridiculous...

**Comment by wilfwilson** [comment]
Indeed even `[2022-01-28T14:18:10.303Z] ['info'] Processing /home/runner/work/Digraphs/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/gaproot/pkg/Digraphs/coverage/oiN94S.coverage.22823...`

**Comment by fingolfin** [comment]
Resolved by #15
